content/en/docs/setup/install/helm/index.md

```diff
@@ -45,62 +27,39 @@ to the missing `istio-token` volume.
    the Istio control plane:
 
     {{< text syntax=bash snip_id=install_base >}}
-    $ helm install istio-base manifests/charts/base -n istio-system
+    $ helm install istio-base istio/base -n istio-system
```

Comment:
```suggestion
    $ helm install istio-base istio/base -n istio-system --set=installCRDs=false
```

Reply:
Can we do it in a followup? Right now this works and is an improvement over the existing model